### PR TITLE
PLANET-7648: Added opt-in archive notice on old posts

### PR DESCRIPTION
**Pull request by mardelnet**
Ref: https://jira.greenpeace.org/browse/PLANET-7648

<hr>

**DESCRIPTION:**
This PR allows website editors to show notices on really old posts so that end users know that the information there may not be relevant anymore. It also includes a migration script to pre-populate the content for this new feature.

Defaults-content-‹-Greenpeace-—-WordPress
Features-‹-Greenpeace-—-WordPress
Mauris-quis-dictum-magna-Greenpeace

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [proteus](https://www-dev.greenpeace.org/test-proteus) | [admin](https://www-dev.greenpeace.org/test-proteus/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-proteus/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-proteus) | [composer-local.json](https://github.com/greenpeace/planet4-test-proteus/blob/main/composer-local.json)

:watch: 2024.11.18 15:38:40

File: assets/src/scss/pages/post/_post.scss [hunk 1]
```diff
@@ -103,3 +103,9 @@
 .related-posts-block {
   margin-bottom: 20px;
 }
+
+.single-post-old-posts-archive-notice {
+  background-color: var(--beige-100);
+  padding: 24px;
+  margin-top: 24px;
```
Comment (comzeradd) [inline assets/src/scss/pages/post/_post.scss:110]: (nit) you can use $sp-3 for both of those

**Review by comzeradd — APPROVED** [review]
Tested locally and code looks good 🌟 

Good to go once it passes UAT.

**Comment by comzeradd** [comment]
Also the unit tests failing, seems to be related to the code introduced in this PR

```
Trying to access array offset on value of type bool

/home/circleci/project/src/Post.php:532
/home/circleci/project/single.php:51
```

**Comment by comzeradd** [comment]
> Also, fixed the problems with the PHP tests. Would you mind re-checking this PR locally once again?

Looks good. Tested locally again after the changes. No issue spotted.

**Pull request merged**